client/src/pages/Search.js

```diff
@@ -121,16 +122,30 @@ class Search extends Page {
 		showSaveSearch: false,
 	}
 
-	getSearchPart(type, query, pageNum = 0, pageSize = 10) {
+	getPaginatedNum = (part, pageNum = 0) => {
+		let goToPageNum = pageNum
+		if (part !== undefined) {
+			goToPageNum = part.pageNum
+		}
+		return goToPageNum
+	}
+
+	getPaginated = type => {
 		const { pagination } = this.props
 		const typeLower = type.toLowerCase()
 		const pageLabel = this.pageLabel(typeLower)
-		const paginatedPart = pagination[pageLabel]
+		return  pagination[pageLabel]
+	}
 
+	pageLabel = (type, prefix = this.componentPrefix) => {
+		return `${prefix}${type}`
+	}
+
```

Comment:
The commit mentions that it improves the paginated labels, but I don't see where that is done.
Also, it mentions it fixes the paginated reports, but what was the problem?

Reply:
This commit should be:

GH-18: Fix pagination in Search and MyReports

Search should use different pagination variables with regard to reports.
MyReports pagination was broken and is fixed by providing the get part with the correct page number.

@gjvoosten @maradragan I'm not sure how to fix the commit message. Should I rebase the commit message and force push?

Reply:
@evdhoudt yes

Reply:
@maradragan I discussed this with @gjvoosten last week and we agreed that rebasing should not be done on this branch anymore. So... the only way to fix the message is to cherry-pick all commits to a new branch and rebase it there... This is a lot of work and I'm not sure if it is worth the effort.